cardinal/plugin_debug.go

```diff
@@ -48,11 +48,11 @@ func (p *debugPlugin) RegisterQueries(world *World) error {
 
 // queryDebugState godoc
 //
-//	@Summary		Get information on all entities and components in world-engine
-//	@Description	Displays the entire game state.
-//	@Produce		application/json
-//	@Success		200	{object}	DebugStateResponse
-//	@Router			/query/debug/state [post]
+//	@Summary      Get all game entities
```

Comment:
The summary and description say the same thing: I'd much rather it be uniform. The description you wrote should be used for summary here.